docs/sphinx/source/contributing.rst

```diff
@@ -460,3 +460,9 @@ If this documentation is unclear, help us improve it! Consider looking
 at the `pandas
 documentation <http://pandas.pydata.org/pandas-docs/stable/
 contributing.html>`_ for inspiration.
+
+Code of Conduct
+~~~~~~~~~~~~~~~
+All contributors are expected to adhere to the `Contrubor Code of Conduct
+<https://github.com/pvlib/pvlib-python/blob/master/CODE_OF_CONDUCT.md#contributor-covenant-code-of-conduct>`_.
+If anyone feels the code has been violated, they can contact the project team.
```

Comment:
If we include a line like this we should provide the admin email address. I hesitate to do that since I don't know if mail is forwarded to me, and I don't check that inbox.

Reply:
https://github.com/pvlib/pvlib-python/blob/master/CODE_OF_CONDUCT.md#enforcement

Reply:
How should we handle this. I didn't want to put the pvlib-admin email address here, because I want people to go to the Code of Conduct and read it. What I don't wan't is people clicking on the email here without having read the Code of Conduct first.

BTW: I believe that emailing the pvlib-admin google group will automatically notify all of us -- I think we tested this out last year? Should we test it again.

Reply:
Since @wholmgren reminded me that the email addresses are in the Code of Conduct, and a paragraph about reporting violations, I think it OK to drop this sentence.